src/warc2zim/content_rewriting/html.py

```diff
@@ -161,3 +139,50 @@ def handle_pi(self, data: str):
 
     def unknown_decl(self, data: str):
         self.handle_decl(data)
+
+    def process_attr(
```

Comment:
no docstring, using indexed values without any information… that's not maintenance-friendly 😉

Reply:
Same for all functions in this file. I've tracked this in https://github.com/openzim/warc2zim/issues/219